src/lucky/support/message_encryptor.cr

```diff
@@ -49,13 +50,33 @@ module Lucky
       iv = value[value.size - @block_size, @block_size]
 
       cipher.decrypt
-      cipher.key = @secret
+      set_cipher_key(cipher)
       cipher.iv = iv
 
       decrypted_data = IO::Memory.new
       decrypted_data.write cipher.update(data)
       decrypted_data.write cipher.final
       decrypted_data.to_slice
     end
+
+    private def set_cipher_key(cipher)
+      begin
+        cipher.key = @secret
+      rescue error : ArgumentError
+        raise InvalidSecretKeyBase.new(<<-MESSAGE
+          It appears the key you set for Lucky::Server.setting.secret_key_base is invalid:
+
+            #{error.message.colorize(:red)}
+
+          Please double check your #{"config/server.cr".colorize(:green)}. You can generate a secret key by entering the following command in your terminal:
+
+            #{"▸".colorize(:green)} lucky gen.secret_key
+        MESSAGE
+        )
```

Comment:
I think starting with a shorter summary helps, also colorizing doesn't work for errors as well since they are in red by default. Thoughts on this?

```suggestion
        raise InvalidSecretKeyBase.new(<<-MESSAGE
          The secret key is invalid:

            #{error.message}

          You can generate a new key using 'lucky gen.secret_key' in your terminal:

             ▸ lucky gen.secret_key
            
          Usually the key is set in 'config/server.cr'
              Lucky::Server.configure do |settings|
                settings.secret_key_base = Lucky::SecretKey.new("your-new-key") 
              end
              
              
          MESSAGE
        )
```

I also usually add a couple newlines at the end to give it some breathing room